CCTPRelayer/script/Upgrade.sol

```diff
@@ -9,12 +9,11 @@ contract AxelarHandlerUpgradeScript is Script {
     CCTPRelayer public relayer;
 
     function setUp() public {
-        vm.createSelectFork(vm.envString("RPC_URL"));
         relayer = CCTPRelayer(payable(0xBC8552339dA68EB65C8b88B414B5854E0E366cFc));
     }
 
     function run() public {
-        vm.startBroadcast(vm.envUint("PRIVATE_KEY"));
+        vm.startBroadcast();
```

Comment:
These environment variables are just not needed and it's easier to run the script without them